src/main/java/com/stripe/param/FileCreateParams.java

```diff
@@ -0,0 +1,253 @@
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+public class FileCreateParams extends ApiRequestParams {
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /**
+   * A file to upload. The file should follow the specifications of RFC 2388 (which defines file
+   * transfers for the `multipart/form-data` protocol).
+   *
+   * <p>Note this field is marked as transient to avoid JSON deserializer. Override
+   * {@link FileCreateParams#toMap()} makes sure that the returned map has this same file instance.
+   */
+  transient Object file;
```

Comment:
Change in params. From the spec, we would accept String value for the file. 
It is `Object` because the builder allows setting `File` and `FileInputStream`